functions/src/index.ts

```diff
@@ -32,6 +37,8 @@ export const client = new MeiliSearch({
 
 const index = client.index(config.meilisearchIndex)
 
+void addSearchableFields()
```

Comment:
Why is there a void in front? Out of curiosity

Reply:
Ahh ! Because I had a lot of trouble with 'Top-level await' on Typescript.
Without the compilator want me to change the `module` for `"esnext"` on `tsconfig.json` which totally breaks my code.